server/itx/heartbeat.go

```diff
@@ -100,5 +105,8 @@ func (h *HeartbeatHandler) Log() {
 		Int("rolldpos-events", numPendingEvts).
 		Str("fsm-state", string(state)).
 		Uint64("height", height).
+		Uint64("actpool-size", numActs).
```

Comment:
How about
`Uint64("actpool-size", h.s.actPool.GetSize()).`

And remove L94-L98